content/master/concepts/packages.md

```diff
@@ -1,513 +1,489 @@
 
-The `spec.controller.permissionRequests` field allows a package author to
-request additional RBAC for the packaged controller. The controller's
-`ServiceAccount` will automatically give the controller permission to reconcile
-all types that its package installs, as well as `Secrets`, `ConfigMaps`, and
-`Events`. Any additional permissions must be explicitly requested.
+```shell {label="helm"}
+helm install crossplane \
+crossplane-stable/crossplane \
+--namespace crossplane-system \
+--create-namespace \
+--set configuration.packages='{xpkg.upbound.io/upbound/platform-ref-aws:v0.6.0}'
+```
 
-> Note that the Crossplane RBAC manager can be configured to reject permissions
-> for certain API groups. If a package requests permissions that Crossplane is
-> configured to reject, the package will fail to be installed.
-> Authorized permissions should be aggregated to the rbac manager clusterrole 
-> (the cluster role defined by the provider-clusterrole flag in the rbac manager) 
-> by using the label `rbac.crossplane.io/aggregate-to-allowed-provider-permissions: "true"`
+### Install offline
 
-The `spec.crossplane.version` field specifies the version constraints for core
-Crossplane that the `Provider` is compatible with. It is advisable to use this
-field if a package relies on specific features in a minimum version of
-Crossplane.
+Crossplane installs packages from a local package cache. By
+default the Crossplane package cache is an 
+[emptyDir volume](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir). 
 
-> All version constraints used in packages follow the [specification] outlined
-> in the `Masterminds/semver` repository.
+Configure Crossplane to use a 
+[PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/)
+to use a storage location containing the Configuration image. Read more about
+configuring the Crossplane Pod settings in the 
+[Crossplane install documentation]({{<ref "../software/install#customize-the-crossplane-helm-chart">}}).
 
-For an example Provider package, see [provider-gcp].
+Provide the name of the Configuration's `.xpkg` file and set 
+{{<hover label="offline" line="7">}}packagePullPolicy: Never{{</hover>}}.
 
-To build a Provider package, navigate to the package root directory and execute
-the following command:
+For example, to install a locally downloaded version of 
+Upbound AWS reference platform set the 
+{{<hover label="offline" line="6">}}package{{</hover>}} to the local filename
+and set the Configuration's
+{{<hover label="offline" line="7">}}packagePullPolicy: Never{{</hover>}}.
 
+```yaml {label="offline"}
+apiVersion: pkg.crossplane.io/v1
+kind: Configuration
+metadata:
+  name: offline-platform-ref-aws
+spec:
+  package: platform-ref-aws
+  packagePullPolicy: Never
 ```
-crossplane build provider
+
+### Installation options
+
+Configurations support multiple options to change configuration package related
+settings. 
+
+
+#### Configuration revisions
```

Comment:
```suggestion
#### Revision History Limit
```

I found a bit confusing not to have titles the same as the option. Given these are subsections under `Installation Options`, I was expecting to see the configuration as is (e.g. `Revision activation policy` instead of `Upgrade policy` or `Skip Dependency Resolution` instead of `Ignore Dependencies`) as the title.

Reply:
> On the other hand, we will have users wanting to accomplish a task but not knowing what Crossplane term to look for (for example Revision History) and will want to search for something more generic or task based (like "Configuration revisions").

I would see the task as "Limiting revision history"; maybe I am biased as someone who knows internals. However, Crossplane terms are also selected with the best effort to reflect the task/configuration, so feel like they are not that 
different than what they are for.

Having a section talking about `Configuration Revisions` (or Package `Revisions` in general) makes sense to me in general, but I find it confusing to have it as a Subsection under `Installation options`. 

Search or "ctrl + f" would work in any case, so I don't think this is something we should consider while looking for a good structure.